### ISSUE TITLE
NVDA is suddenly missing some items in the list of braille displays

### ISSUE THREAD

**Issue by mehgcap** [nvaccess/nvda#12129]
###The Issue:
I recently got a very used Braille Star 80 from Ebay. I connected it to my computer, trying both serial and USB, but NVDA wouldn't connect to it. Soon after I began trying, several entries in the list of supported displays disappeared. Restarting NVDA brought them back one time, but then they vanished again. Ever since, no amount of restarting NVDA or Windows has fixed the problem. HandyTech is among the missing, making it hard to continue to try to connect my Braille Star. My usual display is a BrailleNote MPower, and I'm concerned I won't be able to get that working again. My log shows no errors on startup related to braille or loading drivers.

The current list of display options is as follows:

* Automatic
* BRLTTY
* EcoBraille displays
* hedo MobilLine USB
* hedo ProfiLine USB
* MDV Lilli
* Papenmeier BRAILLEX newer models
* Papenmeier BRAILLEX older models
* Seika braille displays
* no braille

### Steps to reproduce:
Not known, as frustrating as that will be.

### Actual behavior:
BrailleNote, Humanware, Handytech, and some other items are absent from the dropdown list in Settings > Braille > Select Braille Display.

### Expected behavior:
All the usual options should be present.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2020.4

#### Windows version:
Version 2004 (OS Build 19041.804)

#### Name and version of other software in use when reproducing the issue:
N/A, I believe

#### Other information about your system:
braille display in use is a Braille Star 80, firmware 4.31, connected via USB

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### If add-ons are disabled, is your problem still occurring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
yes

**Comment by mehgcap** [comment]
Well, this is just weird. I've disconnected the Braille Star and restarted NVDA. All the missing entries have returned. I have no idea what's going on here, but I thought I'd update this issue just in case this helps.

**Comment by Adriani90** [comment]
Thanks @mehgcap for the update. It might have been a mistake in processing the user specific data. Sometimes it helps to restore the NVDA folder in %appdata%. I am closing this now as works for me. if it occurs again and you can reproduce it, then please comment and we can reopen. Thanks.
